ilastik/app.py

```diff
@@ -140,6 +140,8 @@ def main(parsed_args, workflow_cmdline_args=[], init_logging=True):
 
     _init_threading_logging_monkeypatch()
 
+    _init_preferences()
```

Comment:
I really like the idea of using default directories for storing our files, but I guess I'd find it confusing if ilastik silently moved my files around; I could come looking for `~/.ilastikrc` and just not find it. Maybe we shouldn't do this automatically, or at least ask the user if he's OK with us moving it elsewhere.

Alternatively, as we were discussing, maybe ilastik could just look in multiple places until it finds a config file. It would then log which config file is being used so that no one is surprised that their configs aren't taking effect when there are 2 or more config files in the system